src/components/Map/Map.jsx

```
@@ -2,62 +2,35 @@
-        );
-    });
+    return (
+        <CategoriesProvider>
+
+            {createPortal(<FiltersForm />, filtersPlaceholder)}
+            {createPortal(<MapComponent />, mapPlaceholder)}
+        </CategoriesProvider>
+    );
 };
+
+export const Map = () => {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)